src/app/settings/keyboard-mapping-settings/components/keyboard-mapping-settings-page/keyboard-mapping-settings-page.component.ts

```diff
@@ -126,23 +127,26 @@ export class KeyboardMappingSettingsPageComponent implements OnInit, OnDestroy {
   }
 
   public openCreateKeyboardMapping(): void {
-    this.dialogService.openSidebarDialog(EditKeyboardMappingDialogComponent, (createdKeyboardMapping?: KeyboardMapping) => {
-      if (!createdKeyboardMapping) {
+    this.dialogService.openSidebarDialog(EditKeyboardMappingDialogComponent, (createdKeyboardMappingResponse?: EditKeyboardMappingResponse) => {
+      if (!createdKeyboardMappingResponse || !createdKeyboardMappingResponse.keyboardMapping) {
         return
       }
-      this.actionTriggerService.createActionTrigger(createdKeyboardMapping.actionTrigger).subscribe()
+      this.actionTriggerService.createActionTrigger(createdKeyboardMappingResponse.keyboardMapping.actionTrigger).subscribe()
       this.notificationService.createInfoNotification($localize`keyboard-mapping-settings-page.create-keyboard-mapping.success`)
+      if (!createdKeyboardMappingResponse.shouldClose) {
+        this.openCreateKeyboardMapping()
+      }
     })
   }
 
   public openEditKeyboardMapping(keyboardMapping: KeyboardMapping): void {
     this.dialogService.openSidebarDialog(
       EditKeyboardMappingDialogComponent,
-      (updatedKeyboardMapping?: KeyboardMapping) => {
-        if (!updatedKeyboardMapping) {
+      (updatedKeyboardMapping?: EditKeyboardMappingResponse) => {
```

Comment:
To have the parameter name match the type and be consistent with how similar parameters are named, `Response` should be suffixed, such that it becomes `updatedKeyboardMappingResponse`.

Reply:
Fixed